### PR TITLE
added missing factors and changed defaults

### PR DESCRIPTION
**Pull request by uptonsanderbeck**

### CODE REVIEW

**Comment by sbird** [comment]
Ok! What about the new recombination rate?

**Comment by uptonsanderbeck** [comment]
It should be consistent with whatever the default in rate_network is— which means leaving it unchanged.

File: tools/HeII_input_file_maker.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def a_norm(self, redshift):
 
     def specific_intensity(self, z0, E):
         """Specific intensity based on powerlaw QSO spectrum"""
-        func = lambda z: (self.speed_of_light/(4.*np.pi))*(1./(self.cosmo.Hubble(z)*(1+z)))*(1+z0)**3./((1+z)**3.)*self.a_norm(z)*np.exp(-self.tau(z,z0,E))
+        func = lambda z: (self.speed_of_light/(4.*np.pi))*(1./(self.cosmo.Hubble(z)*(1+z)))*(1+z0)**3./((1+z)**3.)*self.a_norm(z)*E**(-self.alpha_q)*np.exp(-self.tau(z,z0,E))
```
Comment (sbird) [inline tools/HeII_input_file_maker.py:146]: This affects only the qso emissivity history, right?

File: tools/HeII_input_file_maker.py [hunk 2]
```diff
@@ -163,7 +163,8 @@ def _dGamma_dt_int(zz, E):
             return dGamma_int
         #Do the double integral of E from Emax to Elim and zz from redshift to 10.
         w = scipy.integrate.dblquad(_dGamma_dt_int,self.Emax,E_lim, redshift, 10)
-        dGammadt = 4.*np.pi*w[0]*self.eVtoerg*self.cosmo.nHe(redshift)
+        xHeII = np.amax([1 - self.hist.XHeIII(redshift), 0.])
+        dGammadt = 4.*np.pi*w[0]*self.eVtoerg*self.cosmo.nHe(redshift)*xHeII
```
Comment (sbird) [inline tools/HeII_input_file_maker.py:167]: Aha! This is why the heating was weird at late times!

**Comment by sbird** [comment]
ok, perfect!

**Comment by sbird** [comment]
Could you also update the default example HeIIReionizationTable in examples?

**Comment by sbird** [comment]
It can be a separate PR

**Pull request merged**